tasks/copy_to_glacier/schemas/body.json

```diff
@@ -1,154 +1,156 @@
-                  "sizeInBytes": {
-                    "type": "integer",
-                    "description": "Size of the object in bytes"
-                  },
-                  "hash": {
-                    "type": "string",
-                    "description": "Hash of the object from Cumulus"
-                  },
-                  "hashType": {
-                    "type": "string",
-                    "description": "Hash type used to hash the object. Supplied by Cumulus."
-                  },
-                  "version": {
-                    "type": "string",
-                    "description": "Latest version of the file in the S3 Glacier bucket"
-                  },
-                  "ingestTime": {
-                    "type": "string",
-                    "description": "Date and time the file was originally ingested into ORCA.",
-                    "pattern": "^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])T(2[0-3]|[01][0-9]):([0-5][0-9]):([0-5][0-9])(.[0-9]+)?(Z)?\\+00:00$"
-                  },
-                  "etag": {
-                    "type": "string",
-                    "description": "etag of the file object in the AWS S3 Glacier bucket."
-                  }
-                },
-                "required": [
-                  "name",
-                  "cumulusArchiveLocation",
-                  "orcaArchiveLocation",
-                  "keyPath",
-                  "sizeInBytes",
-                  "version",
-                  "ingestTime",
-                  "etag"
-                ]
-              }
-            ]
-          }
+        "shortname": {
+          "type": "string",
+          "description": "Collection short name from Cumulus."
         },
-        "required": [
-          "cumulusGranuleId",
-          "cumulusCreateTime",
-          "executionId",
-          "ingestTime",
-          "lastUpdate",
-          "files"
-        ]
-      }
+        "version": {
+          "type": "string",
+          "description": "Collection version from Cumulus."
+        }
+      },
+      "required": [
+        "collectionId",
+        "shortname",
+        "version"
+      ]
     },
-    "required": [
-      "provider",
-      "collection",
-      "granule"
-    ]
-  }
\ No newline at end of file
+    "granule": {
+      "type": "object",
+      "description": "A dictionary containing granule information",
+      "properties": {
+        "cumulusGranuleId": {
+          "type": "string",
+          "description": "Granule ID from Cumulus"
+        },
+        "executionId": {
+          "type": "string",
+          "description": "AWS step function execution id"
+        },
+        "cumulusCreateTime": {
+          "type": "string",
+          "description": "Date and time data was originally ingested into Cumulus",
+          "pattern": "^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])T(2[0-3]|[01][0-9]):([0-5][0-9]):([0-5][0-9])(.[0-9]+)?(Z)?\\+00:00$"
```

Comment:
Looks like the changes we made to datetime format got lost.

Reply:
https://github.com/nasa/cumulus-orca/pull/172/commits/51eda3c4d144c807da175ef927445b0b2b5ffbbb